setup.py

```diff
@@ -60,7 +45,8 @@ def load_requirements(*requirements_paths):
         'Operating System :: OS Independent',
         'Programming Language :: Python',
     ],
-    packages=PACKAGES,
+    packages=find_packages(exclude=['*.test', '*.tests']),
```

Comment:
Is specifying the exclusion of tests here actually don't anything? I'm guessing not (I think the paths have to be exact). If the exclusion is happening through MANIFEST.in, let's remove it here.